README.md

```diff
@@ -23,6 +23,8 @@ It contains the following packages.
     - `MongoDBAtlasParentDocumentRetriever` - new feature, docs coming soon!
 - Indexing
     - `MongoDBRecordManager` - new feature, docs coming soon!
+- Document loader
+    - `MongoDBLoader` - new feature, docs coming soon!
 - LangGraph persistence
```

Comment:
This feels misleading. Should we use qualified names? I'm for breaking this into sections by package.

Reply:
Yeah, let's do that.  I'll make this PR just about the refactor with a new ticket.  In the changelog PR we can break out the README.